Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    for node in G:
</pre><p>A comment for what's happening here might be nice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>added!</p>